contracts/TWRegistry.sol

```diff
@@ -24,6 +25,7 @@ contract TWRegistry is Multicall, ERC2771Context, AccessControlEnumerable {
     function addModule(address _moduleAddress, address _deployer) external {
         require(hasRole(OPERATOR_ROLE, _msgSender()), "not operator.");
         deployments[_deployer].add(_moduleAddress);
+        deployer[_moduleAddress] = _deployer;
```

Comment:
i wonder if deployer information should lives in the `ThirdwebFactory`
because the Registry is more to keep track of modules that you added to your wallet. The owner can add / remove any deployments into their wallet.
Factory = deal with deployments.